src/Testing/ZopeTestCase/zopedoctest/functional.py

```diff
@@ -237,8 +237,9 @@ def setup_test_class(self):
 
         # If the test_class does not have a runTest method, we add
         # a dummy attribute so that TestCase construction works.
+        # runTest may get called, so set it to something that works.
         if not hasattr(test_class, 'runTest'):
-            setattr(test_class, 'runTest', None)
+            setattr(test_class, 'runTest', lambda x: None)
 
```

Comment:
I and others have removed some `test_suite` functions in Plone without adverse effects the past few years. Whether this works correctly might depend on the test runner that is used.

I hope no one has removed this function and then unnoticedly kept some tests from being discovered. I guess the change is bigger that there *is* a `test_suite` function and that not all test available cases are added to it.